### DISCUSSION TITLE
Including sound classification for SDH subtitles?

### DISCUSSION THREAD

**Discussion by kuroderuta** [Purfview/whisper-standalone-win#252] [Ideas]
SDH stands for subtitles for the deaf and hard of hearing. These subtitles assume the end user cannot hear the dialogue and include important non-dialogue information such as expressions, sound effects or music.

This is both a question if such feature exists and a request if it doesn't. I noticed that sometime Whisper will hallucinate an output like [laugher] but its not consistent and I haven't found the settings to force those. There are also VADs like YAMNet which could perhaps help fill the gap.

**Comment by Purfview** [comment]
Such feature doesn't exist.
Personally I'm not interested in it.
